generator/source/faq/upgrade.rst

```diff
@@ -0,0 +1,53 @@
+---
+layout: page
+hide: true
+tags: [faq]
+title: Sawtooth FAQ - Upgrading Hyperledger Sawtooth
+permalink: /faq/upgrade/
+# Copyright (c) 2019, Intel Corporation.
+# Licensed under Creative Commons Attribution 4.0 International License
+# https://creativecommons.org/licenses/by/4.0/
+---
+Sawtooth FAQ: Upgrading Hyperledger Sawtooth
+============================================
+
+.. class:: mininav
+
+PREVIOUS_ FAQ_ NEXT_
+
+.. contents::
+
+
+Points to consider while upgrading Hyperledger Sawtooth (from version 1.0.X to 1.1.X):
+--------------------------------------------------------------------------------------
+1. Persist the block store, transaction receipt and global state (lmdb files default stored in /var/lib/sawtooth directory).
+2. Persist the keys generated for the validator (default stored in /etc/sawtooth directory).
+3. Hyperledger Sawtooth 1.1.X versions have a major feature introduced where different consensus engine can be plugged into the validator.
+   a. Validator shall open the listener port so that the consensus engine can be registered to it. This in addition to other ports which are open to transaction processor/event listeners/clients, validator connections.
```

Comment:
I would mention that this can be done with the `sawtooth-validator --bind consensus:tcp://localhost:5050` command line option.

Reply:
Done